.travis.yml

```diff
@@ -60,9 +88,21 @@ before_script:
   - cmake --build . --config $EASTL_CONFIG
 
 script:
-    # Run Tests 
-    - cd $TRAVIS_BUILD_DIR/build_$EASTL_CONFIG/test
-    - ctest -C $EASTL_CONFIG -V || exit 1
-    # Run Benchmarks
-    - cd $TRAVIS_BUILD_DIR/build_$EASTL_CONFIG/benchmark
-    - ctest -C $EASTL_CONFIG -V || exit 1
+  # Run the Buck build and tests
+  - buck build :eastl
```

Comment:
rather than doing the buck builds inline with the cmake builds... lets make a seperate travis task for them... similar to to the above for clang3.9